pkg/volume/empty_dir/empty_dir.go

```diff
@@ -107,15 +107,18 @@ func (plugin *emptyDirPlugin) NewMounter(spec *volume.Spec, pod *v1.Pod, opts vo
 
 func (plugin *emptyDirPlugin) newMounterInternal(spec *volume.Spec, pod *v1.Pod, mounter mount.Interface, mountDetector mountDetector, opts volume.VolumeOptions) (volume.Mounter, error) {
 	medium := v1.StorageMediumDefault
+	sizeLimit := &resource.Quantity{}
 
 	if spec.Volume.EmptyDir != nil { // Support a non-specified source as EmptyDir.
 		medium = spec.Volume.EmptyDir.Medium
+		sizeLimit = spec.Volume.EmptyDir.SizeLimit
```

Comment:
need to check for nil? EmptyDirVolumeSource's SizeLimit is `*resource.Quantity`

Reply:
ok